src/OpenDiffix.Core.Tests/QueryEngine.Tests.fs

```diff
@@ -289,17 +289,13 @@ type Tests(db: DBFixture) =
       "SELECT count(*) FROM customers_small GROUP BY city ORDER BY 1"
 
   [<Fact>]
-  let ``Filtering and grouping doesn't change results`` () =
+  let ``Equivalent filtering and grouping doesn't change seed`` () =
     equivalentQueries
-      "SELECT count(*) FROM customers GROUP BY city HAVING city = 'Berlin'"
+      "SELECT count(*) FROM customers WHERE city = 'Berlin' GROUP BY city"
```

Comment:
It is necessary, because when data is filtered out before anonymization there are no outliers to redistribute, resulting in a different output. The same effect happens with LED, so the tested behavior wasn't correct for some time.

Reply:
> when data is filtered out before anonymization there are no outliers to redistribute

just for my understanding - I thought there are outliers, but there aren't other buckets where to distribute them to. Is this what you've meant or am I missing sth?

Other than that, the new form of the assertion seems "easy" to satisfy, so to speak. As in:  the GROUP BY can be ignored by the system under test and we're testing equivalence of identical queries. I'm not sure I'm making a point here.

(I guess we're testing whether the user can't get two samples out of a `GROUP BY` query and a `WHERE` query, correct? BTW Is it possible to break the code in a way that this assertion fails?)

Reply:
OK, this makes sense. OTOH doesn't this (i.e. the offset between the `WHERE` and no-`WHERE` queries) provide a way to reconstruct the flattening?

Reply:
The recovered outliers are themselves anonymized before redistribution. 
One can reconstruct the noisy total value of the outliers, but that information is safe to report.